ohteepee/build.gradle

```
@@ -50,11 +48,10 @@ publishing {
 
 dependencies {
     // Compose
+    api(platform("androidx.compose:compose-bom:$composeBom"))
```

The library should not expose compose dependencies 🙏 

```suggestion
    implementation(platform("androidx.compose:compose-bom:$composeBom"))
```